src/ReactiveSocket.h

```diff
@@ -32,6 +32,8 @@ class KeepaliveTimer {
 
 using CloseListener = std::function<void(ReactiveSocket&)>;
 
+using ResumeSocketListener = std::function<bool(ReactiveSocket& newSocket, const ResumeIdentificationToken& token)>;
```

Comment:
Do we want to expose the ResumeIdentificationToken all the way to the client code or is it just an internal identifier of the socket?